components/BookmarkButton/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import BookmarkPinIcon from "assets/bookmarkicons/BookmarkPinIcon.svg";
+
+export default function BookmarkButton({ isBookmarked, onToggleBookmark }) {
+  return (
+    <>
+      <StyledButton onClick={onToggleBookmark} aria-label="Toggle Bookmark">
+        <StyledBookmarkPinIcon $isBookmarked={isBookmarked} />
+      </StyledButton>
+    </>
```

Comment:
If we are only returning one element from the component, then we don't need the react fragment. 

```suggestion
      <StyledButton onClick={onToggleBookmark} aria-label="Toggle Bookmark">
        <StyledBookmarkPinIcon $isBookmarked={isBookmarked} />
      </StyledButton>
```